src/Components/NumberInputTextField/NumberInputTextField.tsx

```diff
@@ -0,0 +1,26 @@
+import TextField from '@mui/material/TextField'
+import { ChangeEvent } from 'react'
+
+type NumberInputTextFieldProps = {
+    label: string
+    value: number
+    onChangeEvent: (
+        e: ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+    ) => void
+}
+
+export function NumberInputTextField({
```

Comment:
I don't think it's worth it to create a component like this.

It's just casual `TextField` with `type="number"` and `inputProps={{ min: 0}}`.

It just add's more code and more complexity to the app.